src/main/java/org/fundacionjala/at15/katas/pokerhands/agustin/Card.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.at15.katas.pokerhands.agustin;
+
+public abstract class Card {
+    protected String suit;
+    protected String notation;
+    protected int value;
```

Comment:
You could leverage your `Notation` enum here

Reply:
I dont understand what you mean with leverage my enum. You mean to inherit it?

Reply:
Hi Agustin, sorry I missed this question. I mean, you could define this as `protected Notation value` instead of just an int, does it make sense?